PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,13 @@ Checklist:
 <!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->
 
 - [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
-- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`)
+- [ ] I have read [freeCodeCamp's guidelines for creating a Pull Request](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
+- [ ] My pull request has a descriptive title and is less than 30 characters. Good examples are: 
+  - `feat: add more tests to HTML and CSS challenges`
+  - `docs(i18n): Chinese translation of local setup`
```

Comment:
Bad example, that is part of the automated pipelines, translations are not to be done manually

Reply:
These are example provided [in the docs](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) so happy to remove this one, but it should also be removed there as well if it's not a good example.

Reply:
See -> https://github.com/freeCodeCamp/.github/pull/6#issuecomment-1003572213